.github/workflows/codeql.yml

```diff
@@ -0,0 +1,44 @@
+name: "Code Scanning - Action"
+
+on:
+  push:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  CodeQL-Build:
+
+    strategy:
+      fail-fast: false
```

Comment:
I think this task should immediately stop and report fail if something goes wrong, so using `fail-fast: true` might be more appropriate.

Reply:
@Jamminroot we need to get full error list here

Reply:
This one resolved.